### PR TITLE
Add error message if the server is not available

### PR DESCRIPTION
**Pull request by chivchila**
🐿 v2.12.4

### CODE REVIEW

File: src/renderers/search-suggestions.js [hunk 1]
```diff
@@ -41,6 +41,11 @@ class SuggestionList extends BaseRenderer {
 		</li>`;
 	}
 
+	renderErrorMessage(){
+		this.newHtml = `<div class="n-topic-search__error">Search is currently unavailable</div>`;
```
Comment (rowanbeentje) [inline src/renderers/search-suggestions.js]: My only question is whether this dismisses on input blur, like the other interfaces?  It looks like it should...

Comment (i-like-robots) [reply src/renderers/search-suggestions.js]: Possibly one to add to https://github.com/Financial-Times/n-topic-search/issues/25 ?

**Comment by i-like-robots** [comment]
Could you please add a screenshot or capture showing this feature in action?

File: src/renderers/search-suggestions.js (hunk 1 above)
Comment (chivchila) [reply src/renderers/search-suggestions.js]: No interface changes with the new change. Added an optional callback which won't have any implications on current functionality

**Pull request merged**